ide-support/revsblibrary.livecodescript

```diff
@@ -2491,6 +2491,12 @@ command revSBUpdateForDependencies pPlatform, pArchs, pSDK, @xSettings
                xSettings["android,device capabilities"][tFeature]
       end repeat
       
+      local tPlayVersion
+      put tAndroidMetadataA["play"]["version"]into tPlayVersion
```

Comment:
@livecodemichael it looks like if different extensions require different versions of the same dependency that both are included is that right? should we use the semver library here to allow version ranges?